packages/material-ui/src/FormControlLabel/FormControlLabel.js

```diff
@@ -22,6 +22,10 @@ export const styles = theme => ({
       cursor: 'default',
     },
   },
+  /* Styles applied to the root element if `position="start"`. */
+  start: {
```

Comment:
What do you think of?

> The variants applied by and enum property are prefixed, e.g. the colorPrimary class applied by the color="primary" property.

https://material-ui.com/guides/api/#api-design-approach

Reply:
What do you think of renaming `start`, `labelPositionStart`?